### PR TITLE
[Filing Announcement] Filing banner for end of 2019 data collection

### LINKED ISSUE #1653: Filing banner for end of 2019 data collection
Reference: https://github.com/cfpb/hmda-frontend/pull/1220/files

### PR DESCRIPTION
**Pull request by Michaeldremy**
Closes #1653

### CODE REVIEW

File: src/common/constants/dev-config.json [hunk 1]
```diff
@@ -24,11 +24,11 @@
...
+    "heading": "Collection of 2019 HMDA Data Will End",
     "type": "warning",
-    "message": "Resubmissions of 2018 HMDA data will not be accepted after December 31st, 2021.",
+    "message": "Resubmissions of 2019 HMDA data will not be accepted after December 31st, 2022.",
     "link": {
-      "url": "https://ffiec.cfpb.gov/documentation/2021/data-collection-timelines",
+      "url": "https://ffiec.cfpb.gov/documentation/2022/data-collection-timelines",
       "text": "View the HMDA data collection timelines."
     },
     "endDate": "12/31/2021"
```
Comment (meissadia) [inline src/common/constants/dev-config.json:34]: Sorry, I forgot what the current year is 🙈
```suggestion
    "endDate": "12/31/2022"
```

File: src/common/constants/prod-beta-config.json [hunk 2]
```diff
@@ -32,11 +32,11 @@
...
+    "heading": "Collection of 2019 HMDA Data Will End",
     "type": "warning",
-    "message": "Resubmissions of 2018 HMDA data will not be accepted after December 31st, 2021.",
+    "message": "Resubmissions of 2019 HMDA data will not be accepted after December 31st, 2022.",
     "link": {
-      "url": "https://ffiec.cfpb.gov/documentation/2021/data-collection-timelines",
+      "url": "https://ffiec.cfpb.gov/documentation/2022/data-collection-timelines",
       "text": "View the HMDA data collection timelines."
     },
     "endDate": "12/31/2021"
```
Comment (meissadia) [inline src/common/constants/prod-beta-config.json:42]:
```suggestion
    "endDate": "12/31/2022"
```

File: src/common/constants/prod-config.json [hunk 3]
```diff
@@ -24,11 +24,11 @@
...
+    "heading": "Collection of 2019 HMDA Data Will End",
     "type": "warning",
-    "message": "Resubmissions of 2018 HMDA data will not be accepted after December 31st, 2021.",
+    "message": "Resubmissions of 2019 HMDA data will not be accepted after December 31st, 2022.",
     "link": {
-      "url": "https://ffiec.cfpb.gov/documentation/2021/data-collection-timelines",
+      "url": "https://ffiec.cfpb.gov/documentation/2022/data-collection-timelines",
       "text": "View the HMDA data collection timelines."
     },
     "endDate": "12/31/2021"
```
Comment (meissadia) [inline src/common/constants/prod-config.json:34]:
```suggestion
    "endDate": "12/31/2022"
```

**Review by meissadia — CHANGES_REQUESTED** [review]

File: src/common/constants/dev-config.json (hunk 1 above)
Comment (Michaeldremy) [reply src/common/constants/dev-config.json:34]: `endDate` has been updated. 

Banner before user sign in:

image

Banner is still present after user sign in.

File: src/common/constants/prod-config.json (hunk 3 above)
Comment (Michaeldremy) [reply src/common/constants/prod-config.json:34]: endDate has been updated.

**Review by meissadia — APPROVED** [review]
LGTM

**Pull request merged**